pom.xml

```diff
@@ -169,6 +169,33 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <!-- Copy initializer config -->
+            <id>Copy initializer config</id>
```

Comment:
Change this to what the execution actually does...

Reply:
That was not supposed to be in this PR